src/main/kotlin/no/nav/personbruker/dittnav/eventaggregator/kafka/serializer/SwallowSerializationErrorsAvroDeserializer.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.personbruker.dittnav.eventaggregator.kafka.serializer
+
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient
+import io.confluent.kafka.serializers.KafkaAvroDeserializer
+import org.apache.kafka.common.errors.SerializationException
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
```

Comment:
Denne vil bli tatt i bruk i del to av denne feilhåndteringsoppgaven, da det foreløpig mangler en integrasjonstest for at denne fungere som ønsket.